src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/oauth/RemoteAPIAjaxControllerTest.java

```diff
@@ -123,7 +123,7 @@ public Pageable previousPageable() {
 		};
 
 		when(remoteAPIService.search(any(), anyInt(), anyInt(), any(Sort.Direction.class),
-				any(String[].class))).thenReturn(remoteAPIPage);
+				any(String.class))).thenReturn(remoteAPIPage);
```

Comment:
Is there a reason for this change from `String[].class` to `String.class`?

Reply:
`String[].class` appeared to have been an error which was allowed by Mockito 1.x